ember-cli-build.js

```diff
@@ -4,7 +4,19 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
-    // Add options here
+    emotion: {
```

Comment:
Do we need to set this up, or is it just warding against the defaults changing at some point?

Reply:
ember-cli-build is only executes for the dummy app, more or less so we can test options.

Reply:
👍